models/Events.js

```diff
@@ -0,0 +1,87 @@
+const { Schema, model } = require('mongoose');
+
+const eventsSchema = new Schema({
+  name: {
+    type: String,
+    required: [true, 'The name of Event is required'],
+    trim: true
+  },
+  description: {
+    type: String,
+    required: [true, 'The description is required'],
+    trim: true
+  },
+  talks: [{
+    name: String,
+    description: String,
+    date: Date,
+    hour: Date,
+    speakers: [{
+      name: {
+        type:String,
+        require: true,
+        trim: true
+      },
+      avatar: String,
+      twitter: String,
+      biografia: String,
+      rol: String
+    }]
+  }],
+  eventStatus: {
+    type: Boolean,
+    default: false
+  },
+  date: Date,
+  hour: Date,
+  theme: {
+    type: String,
+    // At the moment we accept just 2 omnitrix or cureness!
+  },
+  broadcast: [{
+    subject: {
+      type: String,
+      require: [true, 'the subject is required'],
+      trim: true
+    },
+    text: String,
+    image: String
+  }],
+  sponsors: [{
+    name: String,
+    logo: String,
+    url: String
+  }],
+  countDown: {
+    type: Boolean,
+    default: true
+  },
+  allowRegistre:{
+    type: Boolean,
+    default: true
+  },
+  slug: {
+    type: String
+  },
+  fullUrl: {
+    type: String
+  },
+  organizators: [{
+    name: String,
+    avatar: String,
+    rol: String
+  }],
+  bannerOrHeader: {
+    type: Boolean,

```

Comment:
'type' dont mean anything, a name like 'isBanner' or 'isHeader' is most descriptive